Erro - Teste SEI - Versão  4.1.1  - MYSQL

**Descreva o erro**

teste=TramiteEnvioParcialTest

Mensagem de erro:
O trÃ¢mite ainda nÃ£o foi concluÃ­do. Acompanhe no Painel de Controle o andamento da tramitaÃ§Ã£o, antes de realizar uma nova tentativa. NRE: 0000044253152024. Processo: 99990.000177/2024-01.


....................................EE.........................  63 / 176 ( 35%)
............................................................... 126 / 176 ( 71%)
..................................................              176 / 176 (100%)

Time: 5 hours, Memory: 54.00 MB

There were 2 errors:

1) TramiteEnvioParcialTest::test_criar_documento_processo_recebido_tramitar_destinatario_envio_parcial
PHPUnit\Extensions\Selenium2TestCase\WebDriverException: Timed out after 360 seconds

/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/WaitUntil.php:138
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:597
/tests/tests/CenarioBaseTestCase.php:391
/tests/tests/TramiteEnvioParcialTest.php:135
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:395
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:360
phpvfscomposer:///tests/vendor/phpunit/phpunit/phpunit:97

Caused by
PHPUnit\Framework\ExpectationFailedException: Failed asserting that 'Erro\n
O trâmite ainda não foi concluído. Acompanhe no Painel de Controle o andamento da tramitação, antes de realizar uma nova tentativa. NRE: 0000044106502024. Processo: 99990.000023/2024-01.' contains "Trâmite externo do processo finalizado com sucesso!".

/tests/tests/CenarioBaseTestCase.php:375
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/WaitUntil.php:119
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:597
/tests/tests/CenarioBaseTestCase.php:391
/tests/tests/TramiteEnvioParcialTest.php:135
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:395
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:360
phpvfscomposer:///tests/vendor/phpunit/phpunit/phpunit:97

2) TramiteEnvioParcialTest::test_verificar_processo_recebido_tramitar_remetente_envio_parcial
PHPUnit\Extensions\Selenium2TestCase\WebDriverException: Timed out after 360 seconds

/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/WaitUntil.php:138
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:597
/tests/tests/TramiteEnvioParcialTest.php:164
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:395
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:360
phpvfscomposer:///tests/vendor/phpunit/phpunit/phpunit:97

Caused by
PHPUnit\Extensions\Selenium2TestCase\WebDriverException: no such element: Unable to locate element: {"method":"link text","selector":"99990.000023/2024-01"}
  (Session info: chrome=94.0.4606.61)
  (Driver info: chromedriver=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}),platform=Linux 6.8.0-40-generic x86_64)

/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/Driver.php:168
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/CommandsHolder.php:113
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/Element/Accessor.php:140
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/Element/Accessor.php:178
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/Element/Accessor.php:96
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:440
/tests/src/paginas/PaginaControleProcesso.php:95
/tests/tests/TramiteEnvioParcialTest.php:162
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase/WaitUntil.php:119
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:597
/tests/tests/TramiteEnvioParcialTest.php:164
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:395
/tests/vendor/phpunit/phpunit-selenium/PHPUnit/Extensions/Selenium2TestCase.php:360
phpvfscomposer:///tests/vendor/phpunit/phpunit/phpunit:97

ERRORS!
Tests: 176, Assertions: 2725, Errors: 2.
make: *** [Makefile:195: test-functional] Error 2


-----

**Para reproduzir**
Passos para reproduzir o comportamento:
1. Vá para '...'
2. Clique em '....'
3. Role para baixo até '....'
4. Veja o erro

-----

**Comportamento esperado**
Uma descrição clara e concisa do que você esperava que acontecesse.

-----

**Screenshots**
Se aplicável, adicione capturas de tela para ajudar a explicar seu problema. Sugerimos o uso do Peek (linux) ou ScreenToGif (windows) para captura.

-----

**Contexto da falha (Por favor, complete a seguinte informação):**
 - Versão do sistema: [ex: 1.0.1]
 - Navegador: [ex. chrome, safari]
 - Perfil do Usuário: [ex: colaborador, externo, coordenador, administrador, etc.]
 - Banco de dados: [MySQL, SQLServer, Oracle ou PostgreSQL]
-----

**Informações adicionais**
Adicione qualquer outro contexto sobre o problema aqui.

-----
**Log de Erros**
Adicione o log de erros registrado pelo sistema. Verifique o log registrado no servidor, através da funcionalidade Infra> Log

--- a/tests_sei4/funcional/tests/TramiteEnvioParcialTest.php
+++ b/tests_sei4/funcional/tests/TramiteEnvioParcialTest.php
@@ -63,6 +63,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Verificar processo recebido no Órgão 2 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_criar_processo_contendo_documento_tramitar_remetente_envio_parcial
      * @return void
      */
   public function test_verificar_processo_recebido_tramitar_destinatario_envio_parcial()
@@ -109,6 +110,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Devolver processo ao Órgão 1 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_verificar_processo_recebido_tramitar_destinatario_envio_parcial
      * @return void
      */
   public function test_criar_documento_processo_recebido_tramitar_destinatario_envio_parcial()
@@ -120,12 +122,15 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
       self::$destinatario['SENHA']
     );
 
+    putenv("DATABASE_HOST=org2-database");
+
     $this->paginaBase->navegarParaControleProcesso();
-    $this->paginaControleProcesso->abrirProcesso(self::$protocoloTestePrincipal->getStrProtocoloFormatado());
 
     self::$documentoTeste2 = $this->gerarDadosDocumentoInternoTeste(self::$destinatario);
-    $this->cadastrarDocumentoInterno(self::$documentoTeste2);
-    $this->assinarDocumento(self::$destinatario['ORGAO'], self::$destinatario['CARGO_ASSINATURA'], self::$destinatario['SENHA']);
+    $protocoloTestePrincipalOrg2 = $this->consultarProcessoFixture(self::$protocoloTestePrincipal->getStrProtocoloFormatado(), \ProtocoloRN::$TP_PROCEDIMENTO);
+    $this->cadastrarDocumentoInternoFixture(self::$documentoTeste2, $protocoloTestePrincipalOrg2->getDblIdProtocolo());
+
+    $this->paginaControleProcesso->abrirProcesso(self::$protocoloTestePrincipal->getStrProtocoloFormatado());
 
     $this->tramitarProcessoExternamente(
       self::$protocoloTestePrincipal,
@@ -142,6 +147,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Verificar processo recebido no Órgão 1 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_criar_documento_processo_recebido_tramitar_destinatario_envio_parcial
      * @return void
      */
   public function test_verificar_processo_recebido_tramitar_remetente_envio_parcial()

--- a/tests_sei41/funcional/tests/TramiteEnvioParcialTest.php
+++ b/tests_sei41/funcional/tests/TramiteEnvioParcialTest.php
@@ -63,6 +63,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Verificar processo recebido no Órgão 2 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_criar_processo_contendo_documento_tramitar_remetente_envio_parcial
      * @return void
      */
   public function test_verificar_processo_recebido_tramitar_destinatario_envio_parcial()
@@ -109,6 +110,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Devolver processo ao Órgão 1 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_verificar_processo_recebido_tramitar_destinatario_envio_parcial
      * @return void
      */
   public function test_criar_documento_processo_recebido_tramitar_destinatario_envio_parcial()
@@ -120,12 +122,15 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
       self::$destinatario['SENHA']
     );
 
+    putenv("DATABASE_HOST=org2-database");
+
     $this->paginaBase->navegarParaControleProcesso();
-    $this->paginaControleProcesso->abrirProcesso(self::$protocoloTestePrincipal->getStrProtocoloFormatado());
 
     self::$documentoTeste2 = $this->gerarDadosDocumentoInternoTeste(self::$destinatario);
-    $this->cadastrarDocumentoInterno(self::$documentoTeste2);
-    $this->assinarDocumento(self::$destinatario['ORGAO'], self::$destinatario['CARGO_ASSINATURA'], self::$destinatario['SENHA']);
+    $protocoloTestePrincipalOrg2 = $this->consultarProcessoFixture(self::$protocoloTestePrincipal->getStrProtocoloFormatado(), \ProtocoloRN::$TP_PROCEDIMENTO);
+    $this->cadastrarDocumentoInternoFixture(self::$documentoTeste2, $protocoloTestePrincipalOrg2->getDblIdProtocolo());
+
+    $this->paginaControleProcesso->abrirProcesso(self::$protocoloTestePrincipal->getStrProtocoloFormatado());
 
     $this->tramitarProcessoExternamente(
       self::$protocoloTestePrincipal,
@@ -142,6 +147,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Verificar processo recebido no Órgão 1 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_criar_documento_processo_recebido_tramitar_destinatario_envio_parcial
      * @return void
      */
   public function test_verificar_processo_recebido_tramitar_remetente_envio_parcial()

--- a/tests_super/funcional/tests/TramiteEnvioParcialTest.php
+++ b/tests_super/funcional/tests/TramiteEnvioParcialTest.php
@@ -63,6 +63,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Verificar processo recebido no Órgão 2 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_criar_processo_contendo_documento_tramitar_remetente_envio_parcial
      * @return void
      */
   public function test_verificar_processo_recebido_tramitar_destinatario_envio_parcial()
@@ -109,6 +110,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Devolver processo ao Órgão 1 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_verificar_processo_recebido_tramitar_destinatario_envio_parcial
      * @return void
      */
   public function test_criar_documento_processo_recebido_tramitar_destinatario_envio_parcial()
@@ -120,12 +122,15 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
       self::$destinatario['SENHA']
     );
 
+    putenv("DATABASE_HOST=org2-database");
+
     $this->paginaBase->navegarParaControleProcesso();
-    $this->paginaControleProcesso->abrirProcesso(self::$protocoloTestePrincipal->getStrProtocoloFormatado());
 
     self::$documentoTeste2 = $this->gerarDadosDocumentoInternoTeste(self::$destinatario);
-    $this->cadastrarDocumentoInterno(self::$documentoTeste2);
-    $this->assinarDocumento(self::$destinatario['ORGAO'], self::$destinatario['CARGO_ASSINATURA'], self::$destinatario['SENHA']);
+    $protocoloTestePrincipalOrg2 = $this->consultarProcessoFixture(self::$protocoloTestePrincipal->getStrProtocoloFormatado(), \ProtocoloRN::$TP_PROCEDIMENTO);
+    $this->cadastrarDocumentoInternoFixture(self::$documentoTeste2, $protocoloTestePrincipalOrg2->getDblIdProtocolo());
+
+    $this->paginaControleProcesso->abrirProcesso(self::$protocoloTestePrincipal->getStrProtocoloFormatado());
 
     $this->tramitarProcessoExternamente(
       self::$protocoloTestePrincipal,
@@ -142,6 +147,7 @@ class TramiteEnvioParcialTest extends FixtureCenarioBaseTestCase
      * Verificar processo recebido no Órgão 1 com envio parcial mapeado
      * @group mapeamento
      *
+     * @depends test_criar_documento_processo_recebido_tramitar_destinatario_envio_parcial
      * @return void
      */
   public function test_verificar_processo_recebido_tramitar_remetente_envio_parcial()

